packages/backend/src/services/GitIntegrationService/GitIntegrationService.ts

```diff
@@ -296,7 +296,9 @@ Affected charts:
             );
 
         // Github's path cannot start with a slash
-        const fileName = `${path.replace(/^\//, '')}/${explore.ymlPath}`;
+        const fileName = GitIntegrationService.removeExtraSlashes(
```

Comment:
@rephus fix issue where project path is `/`
<img width="444" alt="Screenshot 2025-02-11 at 19 05 52" src="https://github.com/user-attachments/assets/44569a7c-8136-48ce-b1bb-a95d3b941807" />